tests/Feature/Modules/Invoices/OrderToInvoiceCrudTest.php

```
@@ -0,0 +1,151 @@
+            'state'          => OrderState::SALE,
+            'invoice_status' => InvoiceStatus::TO_INVOICE,
+        ]);
+
+        Order::factory()->count(2)->create([
+            'company_id'     => $this->company->id,
+            'partner_id'     => $this->partner->id,
+            'state'          => OrderState::SALE,
+            'invoice_status' => InvoiceStatus::INVOICED,
+        ]);
+
+        $component = Livewire::actingAs($this->user)
```

PHPMD (2.15.0)

Avoid unused local variables such as '$component'. (undefined)

(UnusedLocalVariable)